### ISSUE TITLE
command line, tx withdraw parse error

### ISSUE THREAD

**Issue by frimin** [pactus-project/pactus#1599]
## Description

withdraw tx command parse error

### How To Reproduce

In 1.6.0  command: pactus-wallet tx withdraw has parse error:

![874fddcc95140a5036d2db08458f77c](https://github.com/user-attachments/assets/8083ff4d-f252-46d1-a5f1-0762ded4084b)

All the same command parameters work fine with 1.5.0

### What Happened

Command line error

### Environment

pactus-wallet 1.6.0

**Comment by alidevjimmy** [comment]
Do you follow cammnd instruction?
```bash
pactus-wallet tx withdraw [flags] <VALIDATOR> <ACCOUNT> <STAKE>
```

**Comment by frimin** [comment]
![image](https://github.com/user-attachments/assets/a0f99bce-a796-4d52-a9de-c49d44fe9240)


The problem can be reproduced in my local network, and there is no exception in version 1.5.0

**Comment by b00f** [comment]
@frimin 

Thank you for reporting this issue. It has been resolved in #1602.

To proceed with your transaction, please specify the fee as follows:

```bash
.pactus-wallet --path ~/pactus/testnet/wallets/default_wallet tx withdraw tpc1p3zhwdr657m9lun8r9fq7z487n6499neh3eexj5 tpc1z2e3xprqdd995yhayacvrfvgfl84zmcdyx8mt5k 1 --fee 1.0
```
